src/components/Home/WalletCard/WalletCard.tsx

```diff
@@ -11,28 +11,43 @@ import {
   convertSatToBtc,
   convertToString,
 } from "../../../util/format";
-import { checkPropsUndefined } from "../../../util/util";
-import LoadingBox from "../../Shared/LoadingBox/LoadingBox";
 
-export const WalletCard: FC<WalletCardProps> = (props) => {
+type Props = {
+  onchainBalance: number | null;
+  lnBalance: number | null;
+  onReceive: () => void;
+  onSend: () => void;
+};
+
+export const WalletCard: FC<Props> = ({
+  onchainBalance,
+  lnBalance,
+  onReceive,
+  onSend,
+}) => {
   const { t } = useTranslation();
   const { unit } = useContext(AppContext);
 
-  if (checkPropsUndefined(props)) {
-    return <LoadingBox />;
+  if (onchainBalance !== null && lnBalance != null) {
   }
 
-  const { onchainBalance, lnBalance } = props;
-
   const convertedOnchainBalance =
     unit === Unit.BTC ? convertSatToBtc(onchainBalance) : onchainBalance;
-  const convertedLnBalance =
-    unit === Unit.BTC ? convertMSatToBtc(lnBalance) : lnBalance / 1000;
+  let convertedLnBalance = null;
+
+  if (lnBalance) {
+    convertedLnBalance =
+      unit === Unit.BTC ? convertMSatToBtc(lnBalance) : lnBalance / 1000;
+  }
+
+  let totalBalance = null;
 
-  const totalBalance =
-    unit === Unit.BTC
-      ? +(convertedOnchainBalance + convertedLnBalance).toFixed(8)
-      : convertedOnchainBalance + convertedLnBalance;
+  if (convertedOnchainBalance != null && convertedLnBalance != null) {
```

Comment:
Not sure why but I always write `!==`. Might be unnecessary in this case.

Reply:
Whoops, my bad. 

Thank you!